notebooks/official/tabular_workflows/tabnet_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1287 @@
+{
```

Comment:
<p>Should <code>dataflow_subnetwork</code> be None instead of ""? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/726/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
Done.